src/docs/release-notes/R22/R22.mdx

```diff
@@ -0,0 +1,60 @@
+### Brightlayer UI Migration
+
+The Power Xpert Blue "PX Blue" design system rebranded to Brightlayer User Interface "Brightlayer UI".
+
+-   [Brightlayer UI](https://brightlayer-ui.github.io/) documentation site
+-   Github organization migrated to [Brightlayer UI](https://github.com/brightlayer-ui)
+-   Component libraries migrated to [Brightlayer UI](https://brightlayer-ui-components.github.io/)
+-   NPM packages migrated to [Brightlayer UI](https://www.npmjs.com/org/brightlayerui)
+
+### Component Libraries
+
+-   Updated Angular Score Card component styles
+-   Updated React Info List Item with conditional title and subTitle rendering
+-   Added React Empty State card variation
+
+### Icons
+
+-   Added 38 new icons
+-   Fixed an icon clipping issue that caused wrong rendering in icon fonts
+
+### Themes
+
+-   Minor updates to Angular themes and dark theme
```

Comment:
According to our change logs, we didn't publish any theme changes last quarter. May want to leave this out.

Reply:
we did but small items

![image](https://user-images.githubusercontent.com/37150565/148825938-a4d0a1fb-ab77-43f6-a1c4-c7492d8328a4.png)

Reply:
Are they merged to master?

Reply:
They are not in master, ok